docs/DEVICE-CONNECTIVITY.md

```diff
@@ -65,9 +65,9 @@ At least one port must be set to be a management port, and that port needs to re
 
 ### Last resort
 
-Unless the `network.fallback.any.eth` configuration item is set to false (as specified in [configuration properties](CONFIG-PROPERTIES.md)), then there is an additional lowest priority item in the list of DevicePortConfigs, based on finding all of the Ethernet and Ethernet-like interfaces (an example of the latter is WiFi and cellular modems) which are not used exclusively by applications. The last resort configuration assumes DHCP and no enterprise proxies.
+If `network.fallback.any.eth` [configuration property](CONFIG-PROPERTIES.md) is set to `enabled` (by default it is disabled), then there is an additional lowest priority item in the list of DevicePortConfigs, called "Last resort" DPC (pubsub key `lastresort`), based on finding all of the Ethernet and Ethernet-like interfaces (an example of the latter is WiFi and cellular modems) which are not used exclusively by applications. The last resort configuration assumes DHCP and no enterprise proxies.
 
-Note that if static IP and/or enterprise proxies are used, it is useful to set `network.fallback.any.eth` to false to avoid having the device try DHCP without proxies when there is some network outage.
+However, even if Last resort is disabled (by default or explicitly by config), it can still be forcefully - albeit only temporarily - used if device finds itself in a situation that it has no source of network configuration available. This, for example, happens when device boots for the very first time, without [bootstrap config](#bootstrap-configuration) or the (legacy) network config override being provided. In that case, device may remain stuck with no connectivity indefinitely (unless the user plugs in a USB stick with a network config later), therefore EVE will try to use Last resort to see if it can provide connectivity with the controller. Once device obtains a proper network config (from the controller or a USB stick), EVE will stop using Last resort forcefully and will keep this network config inside `DevicePortConfigList` only as long as it is enabled explicitly by the config (`network.fallback.any.eth` is `enabled`).
```

Comment:
s/can still be/will still be/ since there is no choice in that case.

Maybe it can be reworded more, since this is unrelated to the setting of the config property.
Something like "However, independent of the above property setting, if the device has no source ... it will use the last resort will be used. For example, this happens when ..."

Reply:
Reworded as suggested.